api/projects/tests/test_views.py

```diff
@@ -157,6 +157,67 @@ def setUp(self):
     def _get_detail_url(self, project_id):
         return reverse("api-v1:projects:project-detail", args=[project_id])
 
+    def test_get_project_data(self):
```

Comment:
This test could be named better, we're listing projects - why not use the word `list` in the test? 

Also, for new tests, we should not add them to existing class based tests unless we really have to. Can we move this to `api/tests/unit/projects/test_unit_projects_views.py` instead?

Reply:
Thanks! moved and renamed properly